client/landing/gutenboarding/utils/domain-suggestions.ts

```diff
@@ -1,28 +1,25 @@
-/**
- * External dependencies
- */
-import { getSuggestionsVendor } from '../../../lib/domains/suggestions';
-
 type DomainSuggestion = import('@automattic/data-stores').DomainSuggestions.DomainSuggestion;
 
 export function getFreeDomainSuggestions( allSuggestions: DomainSuggestion[] | undefined ) {
 	return allSuggestions?.filter( ( suggestion ) => suggestion.is_free );
 }
 
+// FIX ME: this function is duplicated within the domain-picker package
 export function getPaidDomainSuggestions( allSuggestions: DomainSuggestion[] | undefined ) {
 	return allSuggestions?.filter( ( suggestion ) => ! suggestion.is_free );
 }
 
 // Recommend either an exact match or the highest relevance score
+// FIX ME: this function is duplicated within the domain-picker package
 export function getRecommendedDomainSuggestion( allSuggestions: DomainSuggestion[] | undefined ) {
```

Comment:
Should be a selector in the domain store? (@sirreal asked on call)

Reply:
As something that returns some transformed results from the store, this looks like a good candidate for a selector 👍 

Ideally in a separate pre-requisite PR.